warehouse/models/intermediate/gtfs_quality/int_gtfs_quality__scheduled_trip_version_summary.sql

```diff
@@ -16,7 +16,7 @@
 WITH dim_stop_times AS (
     SELECT
         *,
-        CONCAT(arrival_time, departure_time) AS time_pair,
+        CONCAT(arrival_time, " - ", departure_time) AS time_pair,
```

Comment:
nit: whitespace is usually a bit annoying for copy/pasting etc., maybe just hyphens?